fbink_device_id.c

```diff
@@ -346,6 +346,21 @@ static bool
 			strtcpy(deviceQuirks.deviceCodename, "Sangria", sizeof(deviceQuirks.deviceCodename));
 			strtcpy(deviceQuirks.devicePlatform, "Bellatrix4", sizeof(deviceQuirks.devicePlatform));
 			return true;
+		case 0XFA0u:    // KS2
+		case 0XFA1u:
+		case 0XFE5u:
+		case 0XF9Du:
+		case 0XFE4u:
+		case 0XFE3u:
+		case 0X102Eu:
+		case 0X102Du:
```

Comment:
Oh, also, lowercase `0x` (unless clang-format can dealt with that, I don't recall ;p).

Reply:
It seemed to work fine in my Hotfix but sure

Reply:
Oh, it works, it's just a stylistic nitpick ;).

Reply:
Lowercase is objectively better to be fair, I was just using a script to convert the serials that made everything uppercase lol